packages/cli/internal/pkg/cli/workflow/workflow_tasks_test.go

```diff
@@ -200,6 +201,58 @@ func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithTaskNameFailure() {
 	}
 }
 
+func (s *GetWorkflowTasksTestSuite) TestGetWorkflowTasks_WithExitCodeUnknown() {
+	defer s.ctrl.Finish()
+	s.mockProjectClient.EXPECT().Read().Return(s.testProjSpec, nil)
+	s.mockConfigClient.EXPECT().GetUserId().Return(testUserId, nil)
+	s.mockDdb.EXPECT().GetWorkflowInstanceById(ctx.Background(), testProjectName, testUserId, testRunId).Return(ddb.WorkflowInstance{ContextName: testContext1Name}, nil)
+	s.mockCfn.EXPECT().GetStackInfo(testContext1Stack).Return(cfn.StackInfo{Outputs: map[string]string{"WesUrl": testWes1Url}}, nil)
+	s.mockWes.EXPECT().GetRunLog(ctx.Background(), testRunId).Return(wes_client.RunLog{
+		RunId: testRunId,
+		TaskLogs: []wes_client.Log{{
+			Name:      testTaskCompositeName,
+			StartTime: testStartTime.UTC().Format("2006-01-02T15:04:05Z"),
+			EndTime:   testStopTime.UTC().Format("2006-01-02T15:04:05Z"),
+			ExitCode:  nil,
+		}},
+	}, nil)
+
+	tasks, err := s.manager.GetWorkflowTasks(testRunId)
+	if s.Assert().NoError(err) {
```

Comment:
Please remove the if statement as the if isn't needed but the assert is (same as the test below).

Reply:
completed.